test/test-handlebars.js

```diff
@@ -730,6 +730,42 @@ test('reply.view with handlebars engine with partials', t => {
   })
 })
 
+test('reply.view with handlebars engine with partials in production', t => {
```

Comment:
This test was added for covering 
https://github.com/fastify/point-of-view/blob/e32d4ca52ed3c62215d74626b9478855b4c6f559/index.js#L553

but can't find a way to "assert" that this piece
` engine.registerPartial(name, engine.compile(partialsObject[name]))`
has been called other than checking coverage

Reply:
That statement call `getPartials` that call the `lru` cache

So to check it you can use `proxyquire` to mock the `hashlru` dep
Some [example in the org](https://github.com/search?q=org%3Afastify+proxyquire&type=code)

Reply:
That was my idea as well but wanted to confirm, going to do that